src/elfpy/types.py

```diff
@@ -153,11 +153,11 @@ def __str__(self):
                 if value != 0:
                     output_string += f" {key}: "
                     if isinstance(value, float):
-                        output_string += f"{float_to_string(value)}"
+                        output_string += f"{value}"
```

Comment:
If you are leaning away from using this in logs it then I support removing `float_to_string` all-together. I think it's better to use the native fstring formatting when needed anyway. Feel free to mark this as a todo (e.g. in clickup or a github issue) and hold off for a follow-up PR.

Reply:
removed it in this PR (only used in like 2 places still)